index.html

```diff
@@ -167,13 +167,23 @@ <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
                 </div>
                 <div class="col-lg-6 col-md-6 col-sm-12 col-12">
                     <form action="">
-                        <div class="input-group">
+                        <!-- <div class="input-group">
                             <input type="email" class="form-control py-3" placeholder="Enter your Email">
-                            <div class="input-group-append">
+                            <!-- <div class="input-group-append">
                                 <button class="input-group-text subscribe-button text-white px-5 font-weight-bold">
                                     Contact Us
                                 </button>
                             </div>
+                             -->
+                        </div> -->
+                        <input type="text" class="form-control py-3" placeholder="Your Name">
```

Comment:
Refer [bootstrap forms](https://getbootstrap.com/docs/4.3/components/forms/) and the forms  divs using `form-group` classes. Also give some margins it looks a bit tight as there is no space between the container background and input fields.